### PR TITLE
Luis/extend spawn actor

### PR DESCRIPTION
**Pull request by LuisPoveda**
#### Description
Added the optional argument "bone" to the PythonAPI call SpawnActor and TrySpawnActor, this argument takes an string, with the parent socket name where the child actor will be attached.

Tested in Ubuntu 20.04
Python 3.8.10

This PR affects to the PythonAPI so make sure to update the PythonAPI version with
```
make clean
```

```
make LibCarla PythonAPI
```

### CODE REVIEW

**Review by Blyron — APPROVED** [review]

File: LibCarla/source/carla/rpc/Command.h [hunk 1]
```diff
@@ -59,11 +61,17 @@ namespace rpc {
...
+          transform(transform),
+          parent(parent),
+          socket_name(bone) {}
       ActorDescription description;
       geom::Transform transform;
       boost::optional<ActorId> parent;
       std::vector<Command> do_after;
-      MSGPACK_DEFINE_ARRAY(description, transform, parent, do_after);
+      std::string socket_name;
+      MSGPACK_DEFINE_ARRAY(description, transform, parent, do_after, socket_name);
```
Comment (xavisolesoft) [inline LibCarla/source/carla/rpc/Command.h:74]: Note that new command arguments should be also added to the `Command.cpp` file:
https://github.com/carla-simulator/carla/blob/master/PythonAPI/carla/source/libcarla/Commands.cpp#L79

**Review by xavisolesoft — CHANGES_REQUESTED** [review]

File: LibCarla/source/carla/rpc/Command.h [hunk 2]
```diff
@@ -59,11 +61,17 @@ namespace rpc {
         : description(std::move(description)),
           transform(transform),
           parent(parent) {}
+      SpawnActor(ActorDescription description, const geom::Transform &transform, ActorId parent, const std::string& bone)
```
[the remark this answers is not in the document]
Comment (LuisPoveda) [reply LibCarla/source/carla/rpc/Command.h]: Done

File: LibCarla/source/carla/rpc/Command.h (hunk 1 above)
[the remark this answers is not in the document]
Comment (LuisPoveda) [reply LibCarla/source/carla/rpc/Command.h]: Done

Comment (LuisPoveda) [reply LibCarla/source/carla/rpc/Command.h]: Done

[the remark this answers is not in the document]
Comment (LuisPoveda) [reply LibCarla/source/carla/rpc/Command.h]: Tested spawning actor with commands in manual control

**Review by xavisolesoft — APPROVED** [review]

**Pull request merged**